.gitignore

```diff
@@ -0,0 +1,34 @@
+HELP.md
```

Comment:
Thank you for your contribution! I noticed that the `.gitignore` file has been modified in this PR. While I understand this might be helpful for local development, it's usually best practice to keep changes to the `.gitignore` file separate from feature or bug fix PRs. Could you please remove this change from the PR?

Reply:
Sure, I'll definitely fix this. Sorry about that!